crates/services/sync/src/import.rs

```diff
@@ -163,41 +161,23 @@ where
             consensus,
             ..
         } = &self;
-        // Request up to `max_get_header_requests` headers from the network.
-        get_header_range_buffered(range.clone(), params, p2p.clone())
+
+        let headers_res = get_header_range_single_req(range.clone(), p2p.clone()).await;
```

Comment:
We should test what happens when a node tries to fetch a very large range. My guess is that this will exceed the max payload size limit and be rejected, or it could consume a ton of memory if we store 1 million block headers in a vector here. We'll likely need to chunk this range up into separate requests.

Reply:
I'm working on the chunk code rn actually.

Reply:
Maybe we should do a similar strategy to before where we started a stream, but instead of each item being a single block height it would be a sub-range of headers?

Reply:
Hmm. We could do that. What' I've done already is just turn it into a stream of headers like before--the caller can't tell that it's being batched as multiple calls under the hood

It seems to be working. I just had one test that was broken, so I hadn't pushed it up yet.

Reply:
Need to add the batch sizes to config is all now.